br/pkg/utils/retry.go

```diff
@@ -70,3 +82,48 @@ func MessageIsRetryableStorageError(msg string) bool {
 	}
 	return false
 }
+
+// IsRetryableError returns whether the error is transient (e.g. network
+// connection dropped) or irrecoverable (e.g. user pressing Ctrl+C). This
+// function returns `false` (irrecoverable) if `err == nil`.
+//
+// If the error is a multierr, returns true only if all suberrors are retryable.
+func isRetryableError(err error) bool {
```

Comment:
also make the function in `br/pkg/lightning/common/util.go` call this instead. or just remove the copy in Lightning.

we don't want an identical function in two places...

Reply:
fixed